cron/daily-message.js

```diff
@@ -29,17 +30,17 @@ function createMessage({dateString, waitingFor}) {
     }
   }
 
-  if (dayDiff === 0) {
+  if (hourDiff <= 2 && hourDiff >= -2) {
     if (waitingFor.length) {
       return `Today is the last day of release week! We're still waiting on ${waitingFor.join(', ')}. Can we release today?`;
     } else {
       return ':tada: We have released everything on time this week :tada: This is worth a bit of a celebration! :tada: Great work team :muscle:';
     }
   }
 
-  if (dayDiff > 0) {
+  if (hourDiff > 0) {
     if (waitingFor.length) {
-      return `We are currently :rotating_light: ${dayDiff} Days Late :rotating_light: with the release!! We are still waiting on ${waitingFor.join(', ')}`;
+      return `We are currently :rotating_light: ${hourDiff / 24} Days Late :rotating_light: with the release!! We are still waiting on ${waitingFor.join(', ')}`;
```

Comment:
`hourDiff / 24` will need to be either rounded up or down to a non-decimal number.

Reply:
Good catch!